core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -548,7 +592,11 @@ class ColumnFormatRelation(
       df.write.insertInto(snappySession.getIndexTable(indexIdent).toString())
     } catch {
       case e: Throwable =>
-        snappySession.dropTable(indexIdent, ifExists = false)
+        try {
+          snappySession.dropTable(indexIdent, ifExists = false)
+        } catch {
+          case _: Throwable =>
```

Comment:
What is this?? Please never ignore Throwable like this. It hides completely legitimate bugs and also fatal system errors. At most match NonFatal(e).

But why is that required in the first place? If an unexpected exception is coming here then it is a bug in dropTable which should be fixed instead.